connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -164,6 +164,7 @@ object CheckConnectJvmClientCompatibility {
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.streaming.ui.*"),
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.test.*"),
       ProblemFilters.exclude[Problem]("org.apache.spark.sql.util.*"),
+      ProblemFilters.exclude[Problem]("org.apache.spark.sql.scripting.*"),
```

Comment:
Any concerns with this? I think this is all right since we don't need any of the interpreting logic on the client side for Connect. In parallel to this PR, we are testing scripting with Connect e2e and it works without any changes on the client side, especially considering interpreting logic.

Reply:
we should mark it as a private package in https://github.com/apache/spark/blob/master/project/SparkBuild.scala#L328-L336

Reply:
Thanks! Wasn't aware of this...